content/pt-br/community/_index.html

```diff
@@ -2,70 +2,198 @@
+  <img src="/images/community/kubernetes-community-final-05.jpg" alt="Galeria de fotos da conferência do Kubernetes" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Galeria de fotos da conferência do Kubernetes" class="community-gallery-mobile">
+</div>
+
+<div class="community-section" id="values">
+  <h2>Valores da comunidade</h2>
+  <p>Os valores da comunidade do Kubernetes são a pedra fundamental para o sucesso
+    continuado do projeto.<br class="optional"/>
+    Estes princípios guiam todos os aspectos do projeto do Kubernetes.</p>
+  <a href="https://www.kubernetes.dev/community/values/" class="community-cta-button">
+    <span class="community-cta">Leia mais</span>
+  </a>
+</div>
+
+<div class="community-section" id="conduct">
+  <h2>Código de Conduta</h2>
+  <p>A comunidade do Kubernetes valoriza respeito e inclusão e exige o cumprimento
+    do Código de Conduta em todas as interações.</p>
+  <p>Se você testemunhar ou perceber uma violação do Código de Conduta em um evento
+    ou reunião, no <a href="#slack">Slack</a>, ou em outro mecanismo de
+    comunicação, contate o Comitê de Código de Conduta do Kubernetes através do
+    e-mail <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+    Todos os relatos são mantidos em confidencialidade. Você pode ler
+    <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">sobre o comitê</a>
+    no <a href="https://github.com/kubernetes/community">repositório da comunidade do Kubernetes</a>,
+    no GitHub.</p>
+  <a href="/community/code-of-conduct/" class="community-cta-button">
+    <span class="community-cta">Leia mais</span>
+  </a>
+</div>
+
+<div id="videos" class="community-section">
+  <h2>Vídeos</h2>
+
+  <p class="community-simple">
+    O Kubernetes aparece bastante no YouTube. Inscreva-se para ver mais sobre
+    vários tópicos.
+  </p>
+
+  <div class="container">
+    <div class="video youtube">
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg" title="Monthly office hours" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/" ></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg">
+       <span class="videocta">Assista às reuniões de "office hours" mensais&nbsp;&#9654;</span>
+      </a>
+    </div>
+
+    <div class="video youtube">
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ" title="Weekly community meetings" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/"></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ">
+        <span class="videocta">Assista às reuniões da comunidade semanais&nbsp;&#9654;</span>
```

Comment:
Sounds better to me "Assista às reuniões semanais da comunidade". Think about it.

Reply:
Updated!